magpie/setup/magpie-setup-project-spark

```diff
@@ -544,6 +544,21 @@ then
     echo "export SPARK_SSH_OPTS=\"${MAGPIE_REMOTE_CMD_OPTS}\"" >> ${post_sparkenvsh}
 fi
 
+if [ "${HADOOP_SETUP}" == "yes" ] && [ "${SPARK_USE_YARN}" == "yes" ]
+then
+    hadoop_yarn=$(ls "${HADOOP_HOME}"/share/hadoop/yarn/*.jar)
+    for j in ${hadoop_yarn}
+    do
+        sparkclasspath="$j:$sparkclasspath"
+    done
+
+    hadoop_common=$(ls "${HADOOP_HOME}"/share/hadoop/common/*.jar)
+    for j in ${hadoop_common}
+    do
+        sparkclasspath="$j:$sparkclasspath"
+    done
+fi
+
```

Comment:
is every jar really necessary? I suspect it's a much smaller set. LIkely the common client code or something like that.